app/assets/stylesheets/styles/components/crops/components/_crop_info_container.css.scss

```diff
@@ -0,0 +1,46 @@
+// Styles for the crop info container
+
+.crop-info-container {
+  padding: 0;
+
+  .crop-hero {
+    position: relative;
+    height: 300px;
+    background-color: $openfarm-green-muted;
+    background-size: cover;
+    background-position: center;
+    background-image: image-url("leaf-grey-canopy.png");
+
+    .crop-name {
+      position: absolute;
+      margin: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      padding: 50px 10px 10px;
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.